drizzlepac/hapsequencer.py

```diff
@@ -730,7 +751,7 @@ def run_sourcelist_flagging(filter_product_obj, filter_product_catalogs, log_lev
     return filter_product_catalogs
 
 
-def _get_envvar_switch(envvar_name):
+def _get_envvar_switch(envvar_name, default=None):
     """
     This private routine interprets the environment variable, SVM_QUALITY_TESTING,
     if specified.  NOTE: This is a copy of the routine in runastrodriz.py.  This
```

Comment:
Update comment now that multiple env vars are being read.

Reply:
Updated in 65312f8.